validate.lic

```diff
@@ -255,7 +255,7 @@ class DRYamlValidator
     return unless settings.train_workorders
 
     settings.train_workorders
-            .reject { |discipline| %w[Blacksmithing Tailoring Shaping Remedies].include?(discipline) }
+            .reject { |discipline| %w[Blacksmithing Tailoring Shaping Carving Remedies].include?(discipline) }
```

Comment:
Glancing at the code, it doesn't look like Carving is currently supported with `train_workorders` (though it _is_ supported by `disciplines_to_skill` for Traders).